test/unit/math/mix/prob/normal_cdf_test.cpp

```diff
@@ -0,0 +1,28 @@
+#include <test/unit/math/test_ad.hpp>
+#include <limits>
+
+TEST(mathMixScalFun, normal_cdf_derivatives) {
+  auto f = [](const double mu, const double sigma) {
+    return [=](const auto& y) { return stan::math::normal_cdf(y, mu, sigma); };
+  };
+
+  stan::test::expect_ad(f(0.0, 1.0), -50.0);
+  // the following fails because AD returns -nan in the Hessian
```

Comment:
I think delete the commented out tests. We know these things will fail eventually.